### ISSUE TITLE
AsciiMath support

### ISSUE THREAD

**Issue by GrimPixel** [numbas/Numbas#1057]
AsciiMath is easier to type than LaTeX. http://asciimath.org/

**Comment by christianp** [comment]
Are you asking to write AsciiMath in content areas such as the question statement or part prompts?

This would be possible, but we'd have to choose a set of delimiters for AsciiMath. I don't know what would be safe to use with the thousands of existing questions.

To implement this, I think we'd just need to come up with an AsciiMath equivalent of the \var and \simplify LaTeX commands.

**Comment by GrimPixel** [comment]
On the website, it says to use `. It an implementation for VSCode/VSCodium, it is `` for inline or ``` for block:
https://github.com/zmx0142857/vscode-markdown-asciimath

AsciiDoc has a different approach:
https://docs.asciidoctor.org/asciidoc/latest/stem/

**Comment by christianp** [comment]
I don't think it would be safe to use `. That could easily conflict with something else. I'm not sure AsciiDoc's `stem:[...]` syntax would work unambiguously, either.
